examples/orbit.rs

```diff
@@ -6,7 +6,7 @@ use macroquad::prelude::*;
 #[macroquad::main("dolly orbit example")]
 async fn main() {
     // Create a smoothed orbit camera
-    let mut camera = CameraRig::builder()
+    let mut camera = CameraRig::<RightHanded>::builder()
```

Comment:
Still curious why the default works in our own code, but not in `async` context:

```rust
error[E0698]: type inside `async fn` body must be known in this context
  --> examples/orbit.rs:13:10
   |
13 |         .build();
   |          ^^^^^ cannot infer type for type parameter `H`
   |
note: the type is part of the `async fn` body because of this `await`
  --> examples/orbit.rs:53:21
   |
53 |         next_frame().await
   |                     ^^^^^^

error[E0698]: type inside `async fn` body must be known in this context
  --> examples/orbit.rs:25:35
   |
25 |         let camera_xform = camera.update(get_frame_time());
   |                                   ^^^^^^ cannot infer type for type parameter `H` declared on the struct `CameraRig`
   |
note: the type is part of the `async fn` body because of this `await`
  --> examples/orbit.rs:53:21
   |
53 |         next_frame().await
   |                     ^^^^^^

For more information about this error, try `rustc --explain E0698`.
```

Any idea?

Reply:
Async is stressful to the compiler? 🤷😅